tests/nested/core/core-revert/task.yaml

```diff
@@ -33,6 +33,13 @@ execute: |
     #shellcheck source=tests/lib/nested.sh
     . "$TESTSLIB/nested.sh"
 
+    wait_connection_with_store(){
+        while ! execute_remote "sudo snap find test-snapd-tools | grep 'Tools for testing the snapd application'"; do
```

Comment:
you could instead `while ! snap debug connectivity`

Reply:
@sergiocazzolato WDYT about the suggestion from john?